src/builder.rs

```diff
@@ -243,14 +240,14 @@ impl<'req, 'dev, 'win, W: HasRawWindowHandle + HasRawDisplayHandle>
     /// # Errors
     ///
     /// Returns an error when a [`wgpu::Adapter`] cannot be found.
-    async fn build_impl(self) -> Result<Pixels, Error> {
+    async fn build_impl(self) -> Result<Pixels<'win>, Error> {
         let instance = wgpu::Instance::new(wgpu::InstanceDescriptor {
             backends: self.backend,
             ..Default::default()
         });
 
         // TODO: Use `options.pixel_aspect_ratio` to stretch the scaled texture
-        let surface = unsafe { instance.create_surface(self.surface_texture.window) }?;
+        let surface = instance.create_surface(self.surface_texture.window)?;
```

Comment:
This is ... incredible! Does that mean #238 is fixed, and we can add `#![forbid(unsafe_code)]` back to the lib?

Reply:
I'm surprised my approach worked. Since the handle traits have auto-impls for `&W`, I was able to switch from `&'win W` to `W + 'win`, which means you can pass owned objects which satisfy a `+ 'static` bound. This way I was able to overcome the borrow checker for some of the examples by using `Arc`.